kolibri/plugins/facility/assets/src/constants.js

```diff
@@ -32,6 +33,9 @@ export const pageNameToModuleMap = {
   [PageNames.CLASS_ENROLL_LEARNER]: 'classAssignMembers',
   [PageNames.USER_MGMT_PAGE]: 'userManagement',
   [PageNames.FACILITY_CONFIG_PAGE]: 'facilityConfig',
+  [PageNames.DATA_EXPORT_PAGE]: 'dataExport',
```

Comment:
You haven't added vuex modules for these three pages, so there's no need to add these at all. These are only needed when we have a corresponding vuex module for the page in question (we are using this less and less as we rely more and more on composables for handling our shared reactive state).